R/pkg/R/mllib_clustering.R

```diff
@@ -404,11 +411,14 @@ setMethod("summary", signature(object = "LDAModel"),
             vocabSize <- callJMethod(jobj, "vocabSize")
             topics <- dataFrame(callJMethod(jobj, "topics", maxTermsPerTopic))
             vocabulary <- callJMethod(jobj, "vocabulary")
+            trainingLogLikelihood <- callJMethod(jobj, "trainingLogLikelihood")
+            logPrior <- callJMethod(jobj, "logPrior")
```

Comment:
another data point is that `NA` fits in as just another element in vector and list in R but not NULL
```
> as.data.frame(list(1, 2, 2))
  X1 X2 X2.1
1  1  2    2
> as.data.frame(list(1, NULL, 2))
Error in (function (..., row.names = NULL, check.rows = FALSE, check.names = TRUE,  :
  arguments imply differing number of rows: 1, 0
> as.data.frame(list(1, NA, 2))
  X1 NA. X2
1  1  NA  2
```

In Spark SQL, we convert NULL from JVM to NA in DataFrame